config/hsc/jointcal.py

```diff
@@ -7,3 +7,16 @@
 config.applyColorTerms = True
 
 config.colorterms.load(os.path.join(getPackageDir("obs_subaru"), "config", "hsc", "colorterms.py"))
+
+config.sourceSelector.name = 'science'
+config.sourceSelector['science'].doUnresolved = True
+config.sourceSelector['science'].doSignalToNoise = True
+config.sourceSelector['science'].signalToNoise.minimum = 10.
+config.sourceSelector['science'].doIsolated = False
+config.sourceSelector['science'].doFlags = True
+config.sourceSelector['science'].doFluxLimit = True
+config.sourceSelector['science'].fluxLimit.minimum = 10
```

Comment:
Why do you have both `fluxLimit` and `signalToNoise` cuts? I don't see what a minimum flux gains over a SN cut.

Also, I think `fluxLimit` should be named `instFluxLimit`, since that's what it's actually cutting on.

Reply:
Because `signalToNoise` is based on PsfFlux (configurable), and negative `CalibFluxes` were being included and were crashing jointcal.  Remember that?

Reply:
I only vaguely remember it. If that's the reason, we definitely need to document why this is set here. And I wonder if a minimum flux even closer to zero is even more appropriate? Is `1` a better choice than `10`?

Reply:
I just had a thought related to this: if we're using PSF sources now, you should probably also set `config.sourceFluxType="Psf"` so that the `PsfFlux` fields are used instead of `CalibFlux`, no?

Reply:
I'll kick off a rerun as an experiment, but I don't know enough about calibration to assess this.  I've only ever seen aperture fluxes of stars used for calibration.  Paging @erykoff

Reply:
I have been paged!  And I'm confused...I don't know why psf sources are involved in this at all... but to be clear:
a) `slot_CalibFlux` is an alias.  It is by default set to `base_CircularApertureFlux_12_0`, which is what we want to calibrate to.  But if somebody wants to calibrate something else (psf flux?  model flux?), then they can change this.  The photometric calibration code should simply use `calibFlux` and trust the system.
b) If the default for `signalToNoise` is set to `PsfFlux`, this is incorrect if we're using this selection for calibration.  It should be overridden to `CalibFlux` and then we're not mixing in the psf modeling (which is undesirable).  And then if you change the calib flux field, this selection will change with it.
c) Having the `fluxLimit` as well is odd.  Why 10?  Or 1?  Depends on the units, which is strange (but is probably fine for HSC, we know the units, this isn't general).  But if we fix the s/n cut flux field (and we should!), then (I believe) this will become unnecessary.
d) I see John's point about this being `instFluxLimit` in this case, but in general it's just a flux limit and you can set the flux field to anything you want, inst or not.  But again, this is outside the scope of this ticket.

Reply:
You usually use aperture fluxes as they are less subject to systematic noises than PSF fluxes (e.g. brighter fatter).  Because we are calibrating (using many brightish stars) the lower noise of the PSF flux isn't important.